Classes/Image Upload/ImageUploadTableViewController.swift

```diff
@@ -186,13 +196,35 @@ class ImageUploadTableViewController: UITableViewController, UITextFieldDelegate
             }
         }
     }
+    
+}
 
-    // MARK: UITextFieldDelegate
+// MARK: UITextFieldDelegate
 
+extension ImageUploadTableViewController: UITextFieldDelegate {
```

Comment:
Can we actually avoid adding extensions w/in the same file? Trying to keep the code style consistent.

Reply:
I did the same in #746 though, fwiw

Reply:
@rnystrom Sure! But I've seen some style guides recommending this way - https://github.com/raywenderlich/swift-style-guide#protocol-conformance

Reply:
I changed the SLKTextView. There's still the UITextFieldDelegate extension, do you want me to put it without the extension?

Reply:
I actually ran into compilation issues b/c of overuse of `extension` and ObjC interop, so I'm just moving away from them as much as possible 😋 .